.github/workflows/kubernetes-test.yaml

```diff
@@ -0,0 +1,22 @@
+name: Kubernetes Installation Test
+
+on: 
+  pull_request:
+    paths:
+    - 'kubernetes/**'
+  push:
+    branches:
+    - master
+    paths:
+    - 'kubernetes/**'
+
+jobs:
+  kubernetes-installation-test:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Create Kind Cluster
+      uses: helm/kind-action@v1.1.0
+    - name: Install Spilo
+      run: kubectl apply -f kubernetes/spilo_kubernetes.yaml
```

Comment:
Should we have some "test" here in the end maybe to see of Postgres is up? e.g. query pods with master label?

Reply:
I don't think this is necessary. We Already exercise Spilo in many different places on K8s.
This manifest is just for people who want to deploy it manually.

Reply:
The main purpose of this test is to make sure that at least "kubectl apply" is successful (deprecated APIs are not used, etc.).
If the manifest file will be maintained in the future, it is good to have the certainty of apply guaranteed.

I used kind because that's the most reliable way, but I agree that it can be misleading that this is an spilo E2E test (my naming may be inappropriate).
If I should limit the scope to detecting deprecated APIs, other tools may be better ([kubeent](https://github.com/doitintl/kube-no-trouble)?).